compiler/quilt/tools/command.py

```diff
@@ -394,12 +394,22 @@ def _check_team_login(team):
                 "Can't log in as a public user; log out from team %r first." % existing_team
             )
 
+team_regex = re.compile('^[a-z]+$')
+def _check_team_id(team):
+    if team is not None and team_regex.match(team) is None:
+        raise CommandException(
+            "The team you specified is not a valid team. Teams are specified " +
```

Comment:
Brevity is clarity: `Invalid team name: {team}. Lowercase letters only.`

Reply:
It's critical to report back the invalid name so people know where to fix the error.